parsers/AR.py

```diff
@@ -28,16 +23,16 @@
 
 def fetch_production(
     zone_key="AR",
-    session=None,
-    target_datetime: datetime.datetime = None,
-    logger: logging.Logger = logging.getLogger(__name__),
+    session: Union[Session, None] = None,
+    target_datetime: Union[datetime, None] = None,
```

Comment:
As mentioned in the other PR: could we use [typing.Optional](https://docs.python.org/3/library/typing.html#typing.Optional) here? I find `Optional[datetime]` cleaner than `Union[datetime, None]`.

Reply:
Yeah that works, I should actually update the other PR with that too..